apps/docs/content/troubleshooting/premature-client-termination.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title = "Supavisor client terminates connection prematurely"
+topics = ["database", "supavisor"]
+keywords = ["connections", "pooler"]
+
+[[errors]]
+message = "Terminating with reason :client_termination when state was :busy"
+---
+
+This error happens when the client terminates the connection before the connection with the database is completed.
```

Comment:
Is there any suggested resolution? What should the user do?

Reply:
actually, this happens when the client sends a query but terminates the connection with supavisor before receiving a response